src/sage/schemes/elliptic_curves/ell_curve_isogeny.py

```diff
@@ -3719,6 +3719,7 @@ def compute_intermediate_curves(E1, E2):
     EXAMPLES::
 
         sage: from sage.schemes.elliptic_curves.ell_curve_isogeny import compute_intermediate_curves
+
         sage: E = EllipticCurve(GF(83), [1,0,1,1,0])
```

Comment:
Why space here?

As far as I know, this is not a convention.

Reply:
This change was a leftover from marking the following block with a `# needs` that is not needed any more. 
I've removed the blank line again in 2a3f2bf5258